custom_components/unifiprotect/binary_sensor.py

```diff
@@ -189,13 +241,7 @@ def device_class(self):
     @property
```

Comment:
You should be able to drop `device_class` now since hass will get it from `entity_description`